src/components/request/RequestHero.tsx

```diff
@@ -31,14 +32,25 @@ const RequestHero: FC<Props> = ({ chainId }) => {
     chainName = CHAINS[chainId].name;
   }
   const { oracle, state } = useClient();
-  const { requestState, parsedIdentifier } = useReader(state);
+  const { requestState, parsedIdentifier, requestTx, explorerUrl } =
+    useReader(state);
 
   return (
     <HeroSection>
       <HeroContentWrapper>
         <HeroHeaderRow>
           <HeaderTitle>
             {parsedIdentifier || "Optimistic Oracle Request"}
+            <RequestTxText>
+              Request:{" "}
+              <a
+                target="_blank"
+                rel="noreferrer"
+                href={`${explorerUrl}/tx/${requestTx}`}
```

Comment:
this explorer string is already exported from the reader.

Reply:
So I just need to pass in exploreRequestTx ?